packages/rpc-core/src/index.ts

```diff
@@ -14,6 +14,7 @@ import { GetSlotApi } from './rpc-methods/getSlot';
 import { GetStakeMinimumDelegationApi } from './rpc-methods/getStakeMinimumDelegation';
 import { GetTransactionCountApi } from './rpc-methods/getTransactionCount';
 import { MinimumLedgerSlotApi } from './rpc-methods/minimumLedgerSlot';
+import { GetLatestBlockhashApi } from './rpc-methods/getLatestBlockhash';
```

Comment:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/13243/235497687-a3e5a250-b8da-4593-a6ad-45455728be78.png">

This is on me. I need to add the `sort-imports` linter upstream.